ramlfications/utils.py

```diff
@@ -54,24 +56,39 @@ def load_schema(data):
     return data
 
 
-def requests_download_xml():
+def requests_download(url):
```

Comment:
I don't the `requests` package can actually "download" a local file, unlike urllib.  Please double check - but I think you'll have to force `urlib_download` for files.